package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cubehash.js",
   "description": "CubeHash is a very simple cryptographic hash function",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Never make version changes on branches. When merged into master use `npm version patch` to get the sae functionality, but also have the version bump in a separate commit with a git tag as well